part-4/building-block-concepts.md

```diff
@@ -58,42 +75,32 @@ MoBi® internally works with amounts rather than concentrations. Hence, all conc
+Events describe conditional changes during a simulation, usually a time- dependent change of values or formulas, but also complex events like drug administration, nutritional uptake, or physical exercise. MoBi®allows for any kind of change which can be described by means of an `if then else` condition.
 
 {% hint style="tip" %}
-An event is explicitly defined by a boolean formula including an "if" condition, which is will very often depend on time, specific parameters or the amount or concentration of a specific molecule.
+An event is explicitly defined by a boolean formula including an `if` condition, which often depends on time, specific parameters, or the amount or concentration of a specific molecule.
 {% endhint %}
 
 An important distinction is whether an event only occurs once or repeatedly whenever the condition is true.
 
-## Simulation Settings‌
+## Initial Conditions‌
 
-The simulation settings can be managed separately from the actual simulations and can hence be used in different simulations in a project. When a new project is created, a default simulation settings building block is created. Simulation settings are divided into four distinct groups:
+This building block defines the start values (either as a constant value or a formula) of the molecules in the model. Additionally, Initial Conditions BB defines the containers in which the molecules will be present and whether their values can become negative.
 
-- Output Intervals: start and end time point of a simulation and resolution of a simulation; can be edited in this view in the **Output Intervals** tab.
+## Parameter Values‌
 
-- Solver Settings: solver parameters such as tolerance, use of Jacobian etc.; can also be edited in this view in the **Solver Settings** tab.
+The **Parameter Values (PV)** building block defines the values (constant or formula) of the parameters in the model. Values in the PV BB always overwrite the default values defined in other building blocks such as the spatial structure or the molecules building block. Therefore, the PV BB should only contain values for parameters that are different from the default values.
 
-- Output Selection: outputs that will be available for plots; cannot be edited in this view but will be listed in the **Output Selection** tab after a successful simulation run in the **Simulation Creation Wizard**.
+## Individuals‌
 
-- Chart Templates: a set of chart templates can be managed in the **Chart Editor** as described in [Tools](tools.md); is part of the simulation settings, but not visible in this view.
+The **Individuals** building block defines the physiological properties of an individual. The parameter set referred to is limited to the parameters provided by PK-Sim®. Technically, the Individuals BB is comparable to the Parameter Values BB with additional metadata. New individuals can be created in MoBi® the same way as in PK-Sim®.
```

Comment:
_Technically, the Individuals BB is comparable to the Parameter Values BB with additional metadata._
Not sure if I would keep this sentence. Ofc they are comparable, but there are so many differences (Individuals can add parameters to the structure, modified values are only written into PV in commit to BBs etc.)

Reply:
Really? Seems I urgently need to learn MoBi 😆